src/os/macosx/font_osx.cpp

```diff
@@ -298,6 +298,40 @@ const Sprite *CoreTextFontCache::InternalGetGlyph(GlyphID key, bool use_aa)
 	return new_glyph.sprite;
 }
 
+static CTFontDescriptorRef LoadFontFromFile(const std::string &font_name)
+{
+	if (!MacOSVersionIsAtLeast(10, 6, 0)) return nullptr;
```

Comment:
We haven't support 10.6 or older in years. Do we need these checks?

I think we currently aim 10.13.

Reply:
Well, I just kept the test that was already in `LoadCoreTextFont`.